.github/workflows/codeql-analysis-linux.yml

```diff
@@ -0,0 +1,48 @@
+name: "CodeQL for Linux"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '23 11 * * 6'
+
+jobs:
+  analyze-linux:
+    name: Analyze Linux (Ubuntu 18.04)
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'cpp' ]
+        #language: [ 'cpp', 'javascript', 'python' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
+        # Learn more:
+        # https://docs.github.com/en/free-pro-team@latest/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#changing-the-languages-that-are-analyzed
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install Qt
+      run: |
+        sudo apt-get update
```

Comment:
In my script I did an upgrade too. Not sure if this is needed.

Reply:
No, I had the upgrade and it took ages on every build.  So I took it out and it still worked fine.

Reply:
Yes. Same issue here. I will remove it from my pr too.